sql-cli/noxfile.py

```diff
@@ -23,16 +23,27 @@ def dev(session: nox.Session) -> None:
 
 
 @nox.session(python=["3.7", "3.8", "3.9"])
-def test(session: nox.Session) -> None:
+@nox.parametrize("airflow", ["2.2.5", "2.4"])
+def test(session: nox.Session, airflow: str) -> None:
     """Run both unit and integration tests."""
     session.install("-e", "../python-sdk/.[all]")
     session.install("-e", "../python-sdk/.[tests]")
     session.install("poetry")
-    session.run("poetry", "install")
+    session.run("poetry", "install", "--with", "apache_airflow_providers_amazon")
```

Comment:
Why only this provider? We should add all here.

Btw I think this is confusing that we have nox. Nox is currently not in our dependencies of sql-cli. We only use it in CI.

Reply:
:+1: it would be great if we could centralize the dependencies management using Poetry

Reply:
Removed this, not needed anymore since we're mocking the connection with Redshift.